src/Microsoft.AspNetCore.Mvc.Core/ModelBinding/CompositeValueProvider.cs

```diff
@@ -17,11 +21,25 @@ public class CompositeValueProvider :
         IEnumerableValueProvider,
         IBindingSourceValueProvider
     {
+        private readonly ILoggerFactory _loggerFactory;
+        protected readonly ILogger logger;
```

Comment:
No `protected` fields and definitely no fields with incorrect names. `private readonly ILogger _logger`.